src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -25,6 +25,8 @@ type Props = {
 };
 
 const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Props) => {
+    const MAX_VERSIONS = 1000;
```

Comment:
Should this value be shared by `src/api/Versions:getVersions` somehow? If not, can it be moved out of the function to avoid be redeclared?